src/index.js

```diff
@@ -0,0 +1,96 @@
+// change the scene depending on the sky
+// make a reset city name button
+// add a "landscape" element
+
+const state = {
+  clickCount: 50,
```

Comment:
Be sure to use names that reflect the meaning of the value. So here, `temperature` would be a good choice.